packages/genie/package.py

```diff
@@ -88,27 +88,14 @@ def url_for_version(self, version):
     def patch(self):
         filter_file(r'-lnsl','','src/make/Make.include')
 
-    @property
-    def build_targets(self):
-        cxxstd = self.spec.variants["cxxstd"].value
-        cxxstdflag = (
-            "" if cxxstd == "default" else getattr(self.compiler, "cxx{0}_flag".format(cxxstd))
-        )
-        args = [
-            "GOPT_WITH_CXX_USERDEF_FLAGS=-g -fno-omit-frame-pointer {0}".format(cxxstdflag),
-            "all",
-        ]
-        return args
-
     def configure_args(self):
         args = [
             "--enable-rwght",
             "--enable-fnal",
             "--enable-atmo",
             "--enable-event-server",
             "--enable-nucleon-decay",
-            "--enable-neutron-osc",
-            "--enable-vle-extension",
+            "--enable-nnbar-oscillation",
```

Comment:
Is this change appropriate for all GENIE versions supported by this recipe?

Reply:
yes – these flags were changed in the upgrade from v2 to v3, and 3.0.0 is the earliest version supported in this recipe.